docs/dataset/advanced-usage/dataset-formatting/computer-vision.md

```diff
@@ -14,19 +14,19 @@ The Kolena SDK supports upload of data in the Pandas [`DataFrame`](https://panda
 
 The Kolena web app supports the following file formats.
 
-| Format    | Description                                              |
-|-----------|----------------------------------------------------------|
-| `.csv`     | Comma-separated values file, ideal for tabular data.     |
-| `.parquet` | Apache Parquet format, efficient for columnar storage.   |
-| `.jsonl`   | JSON Lines format, suitable for handling nested data.    |
+| Format | Description |
+| --- | --- |
+| `.csv` | Comma-separated values file, ideal for tabular data |
+| `.parquet` | Apache Parquet format, efficient for columnar storage |
+| `.jsonl` | JSON Lines format, suitable for handling nested data |
 
 Supported file types are:
 
-| Type    | Format                                              |
-|-----------|----------------------------------------------------------|
-| Images     | `jpg`, `jpeg`, `png`, `gif`, `bmp` and other web browser supported images|
-| Video | `mov`, `mp4`, `mpeg`, `avi` and other web browser supported video types.   |
-| Point Cloud   | `.pcd`    |
+| Type | Format |
+| --- | --- |
+| Images | `jpg`, `jpeg`, `png`, `gif`, `bmp` and other web browser supported images |
+| Video | `mov`, `mp4`, `mpeg`, `avi` and other web browser supported video types |
```

Comment:
seems like we forgot to remove the `avi` at here https://github.com/kolenaIO/kolena/pull/674
would you mind removing it in this pr?